src/scripts/quests/questTypes/CaptureSpecificPokemonQuest.ts

```diff
@@ -22,9 +22,9 @@ class CaptureSpecificPokemonQuest extends Quest implements QuestInterface {
             return this.customDescription;
         }
         if (this.amount === 1) {
-            return `Capture ${this.pokemon.name}.`;
+            return `Capture or hatch ${this.pokemon.name}.`;
         }
-        return `Capture ${this.pokemon.name} ${this.amount} times.`;
+        return `Capture or hatch ${this.pokemon.name} ${this.amount} times.`;
```

Comment:
`or hatch` is dependent on the includeBreeding parameter, you'll need to make the class remember it (in a new property), and reference here with
```ts
        return `Capture ${this.includeBreeding ? 'or hatch ' : ''}${this.pokemon.name} ${this.amount} times.`;
```

Reply:
Incorporated in now.